cmd/kubeadm/app/version/base.go

```diff
@@ -1,63 +0,0 @@
-/*
```

Comment:
does kubeadm want to use component-base? One decision originally was that component-base targets our daemons. Clis can use it, but were not the focus.

Reply:
Yes, we already pull that as a dependency (because of the feature gates and `NewMapStringString`).
I am perfectly fine with removing the kubeadm copy of the version package and using the one from component-base instead.